### DISCUSSION TITLE
why there is no Multiple User options for github desktop?

### DISCUSSION THREAD

**Discussion by faheemstepsharp** [community/community#136028] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I want to use github desktop for personnel and also for company but cant

**Comment by romhenri** [comment]
In fact, there is no such thing as pure.

But you can make a quick switch by clicking on your photo in the name commit space.

**Comment by steveward** [comment]
@faheemstepsharp thanks for the question. This is being tracked in https://github.com/desktop/desktop/issues/3707, you can follow along there for updates.
